examples/tracing-jaeger/src/main.rs

```diff
@@ -17,12 +17,10 @@ fn init_tracer_provider() -> Result<opentelemetry_sdk::trace::TracerProvider, Tr
 
     Ok(TracerProvider::builder()
         .with_batch_exporter(exporter, runtime::Tokio)
-        .with_config(
-            sdktrace::Config::default().with_resource(Resource::new(vec![KeyValue::new(
-                SERVICE_NAME,
-                "tracing-jaeger",
-            )])),
-        )
+        .with_resource(Resource::new(vec![KeyValue::new(
```

Comment:
we could potentially offer an easier ways for users who is just trying to add service name. For another time!